libbpf-cargo/src/build.rs

```diff
@@ -297,6 +299,21 @@ pub fn build_single(
         options.to_string()
     };
 
+    if let Some(vmlinux_h) = vmlinux_h {
+        let file = fs::File::create(vmlinux_h)?;
+        let bpftool = extract_command_or_default(bpftool, "bpftool");
+        Command::new(bpftool)
+            .arg("btf")
+            .arg("dump")
+            .arg("file")
+            .arg("/sys/kernel/btf/vmlinux")
+            .arg("format")
+            .arg("c")
+            .stdout(file)
+            .status()?;
+        compiler_options += &format!(" -I{}", vmlinux_h.parent().unwrap().display());
```

Comment:
Let's not panic here and instead handle the potential error gracefully, please.

Reply:
Done